gno/r/govdao/seed.gno

```diff
@@ -0,0 +1,104 @@
+package govdao
+
+var UserTiers = map[string]string{
+	"moul":                 "T1",
+	"jaekwon":              "T1",
+	"thehowl":              "T1",
+	"zivkovicmilos":        "T1",
+	"leohhhn":              "T1",
+	"gfanton":              "T1",
+	"harry-hov":            "T1",
+	"deelawn":              "T1",
+	"ajnavarro":            "T1",
+	"ltzmaxwell":           "T1",
+	"tbruyelle":            "T2",
+	"r3v4s":                "T2",
+	"notJoon":              "T2",
+	"albttx":               "T2",
+	"omarsy":               "T2",
+	"piux2":                "T2",
+	"aeddi":                "T2",
+	"jefft0":               "T2",
+	"n0izn0iz":             "T2",
+	"alexiscolin":          "T2",
+	"mvertes":              "T2",
+	"petar-dambovaliev":    "T2",
+	"Villaquiranm":         "T2",
+	"grepsuzette":          "T2",
+	"odeke-em":             "T2",
+	"sw360cab":             "T2",
+	"linhpn99":             "T2",
+	"waymobetta":           "T2",
+	"jeronimoalbi":         "T2",
+	"kazai777":             "T2",
+	"hthieu1110":           "T2",
+	"stefann-01":           "T2",
+	"Kouteki":              "T2",
+	"MikaelVallenet":       "T2",
+	"MalekLahbib":          "T2",
+	"schollz":              "T2",
+	"agherasie":            "T2",
+	"kristovatlas":         "T2",
+	"n2p5":                 "T2",
+	"faddat":               "T2",
+	"x1unix":               "T3",
```

Comment:
we should remove T3 here as they must be elected with invitations
sorry for the bad initial request

(see Mikael's comment in `govdao.gno`)

Reply:
@n0izn0iz done here ad4fdefeb5f8c4c3e5b3a8e82c5e6b288772822b
and for the generate script:
https://github.com/samouraiworld/gnolove/pull/9/commits/f5bee3d2ece2f78d653f6f4f3dbf09445e59ae08